server/routes/termdb.violin.ts

```diff
@@ -66,8 +66,7 @@ function init({ genomes }) {
 			const g = genomes[req.query.genome]
 			const ds = g.datasets[req.query.dslabel]
 			if (!g) throw 'invalid genome name'
-			const data = await trigger_getViolinPlotData(req.query, null, ds, g) // as getViolinResponse
-			res.send(data)
+			await trigger_getViolinPlotData(req.query, res, ds, g) // as getViolinResponse
```

Comment:
@karishma-gangwani `res.send(...)` should not have been deleted, let me know if you saw an issue that required deleting that